release-1.8/release_notes_draft.md

```diff
@@ -431,12 +453,32 @@ kind.
 
 ### Auth
 
+#### Features
+
 * [GA] The RBAC API group has been promoted from v1beta1 to v1. No API changes were introduced.
 * [beta] Advanced auditing has been promoted from alpha to beta. The webhook and logging policy formats have changed since alpha, and may require modification.
 * [beta] Kubelet certificate rotation through the certificates API has been promoted from alpha to beta. RBAC cluster roles for the certificates controller have been added for common uses of the certificates API, such as the kubelet's.
 * [beta] SelfSubjectRulesReview, an API that lets a user see what actions they can perform with a namespace, has been added to the authorization.k8s.io API group. This bulk query is intended to enable UIs to show/hide actions based on the end user, and for users to quickly reason about their own permissions.
 * [alpha] Building on the 1.7 work to allow encryption of resources such as secrets, a mechanism to store resource encryption keys in external Key Management Systems (KMS) was introduced. This complements the original file-based storage and allows integration with multiple KMS. A Google Cloud KMS plugin was added and will be usable once the Google side of the integration is complete.
 
+#### Other features and bug fixes
```

Comment:
we don't have this for any other SIG (so far at least!?)

Reply:
All the SIGs are using different formats at the moment.

Reply:
Indeed :(
Hopefully that will be addressed by the relnotes owners later